app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 27
-    buildToolsVersion "27.0.2"
```

Comment:
This change is not necessary for now. When we need to update libraries, we would do that in a separate task